.github/pull_request_template.md

```diff
@@ -1,8 +1,13 @@
-<!-- 
+STOP!!!
 
-This website is built upon Lab Website Template.
-Please read its documentation carefully before submitting your changes:
+You are about to open this pull request against THE TEMPLATE ITSELF. You probably meant to open it against your own website repo.
```

Comment:
How does this work only in the template repo? It seems like this would also show up whenever someone did a PR against their own repo. This must be some magic I don't understand.